app/assets/stylesheets/bootstrap_and_overrides.css.scss

```diff
@@ -0,0 +1,13 @@
+.bootstrap {
```

Comment:
Yes, they were just remedies to conflicts, I'll get rid of everything thanks to your PR.